service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NotifyGatekeeperController.java

```diff
@@ -51,18 +52,29 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStartEvent(@RequestBody
         return respond(caseDetails, errors);
     }
 
+    @PostMapping("/about-to-submit")
+    public AboutToStartOrSubmitCallbackResponse handleAboutToSubmitEvent(@RequestBody CallbackRequest callbackRequest) {
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        CaseDetails caseDetailsBefore = callbackRequest.getCaseDetailsBefore();
+        CaseData caseData = getCaseData(caseDetails);
+        CaseData caseDataBefore = getCaseData(caseDetailsBefore);
+
+
+        if (SUBMITTED.equals(caseDataBefore.getState())) {
```

Comment:
maybe safer is not to have this check at all. this callback is called only from sendToGatekeeper and not from notifyGatekeeper. and sendToGatekeeper in ccd def is configured to be in single state - submitted. 

On the other side having check is safer as if somebody change ccd def in future to be accessible from different state then restriction will be in place.

Reply:
Moved to a new controller 'SendToGatekeeperController' since it's not shared between both events.